test/browser/features/navigation-changes.feature

```diff
@@ -1,3 +1,5 @@
+# Skipped on Edge 80 pending PLAT-14141
+@skip_edge_80
```

Comment:
This work uncovered a bug in Maze Runner that meant we weren't actually running tests on Edge 80 at all.  Having resolved that I found these scenerios are failing and need further investigation.